week2/lab11/app.js

```diff
@@ -74,10 +74,13 @@ function displayChart(){
   for (var i = 0; i < images.length; i++) {
     s += ' ' + images[i].src + ' had ' + images[i].Nclicks + ' click(s) and was shown ' + images[i].Nshown + ' time(s).' + '</br>';
   }
-  buttonContainer.style.display = 'none';
+  buttonContainer.style.display = 'block';
+  // block or none.
   gebi('totalVotes').innerHTML = s;
   draw(images, imgNames);
   newRoundBtn.style.visibility = 'visible';
+  localStorage.chartVisible = true;
```

Comment:
you need to set chartVisible to false when it's not visible, otherwise this will always be true